Apr 19, 2020 3:54:28 PM net.jini.lookup.ServiceDiscoveryManager lookup
INFO: Exception occurred during query, discarding reggie [com.sun.jini.reggie.RegistrarProxy[registrar=bdd1abdf-ef92-4c2f-b5f9-208cdf9c1a31 DynamicSmartStub [ImplObjClass: com.gigaspaces.grid.lookup.ServiceGridRegistrar, ConnectionURL: NIO://YoramMacBookPro.local:53965/pid[63802]/1026921312924794_2_-519568947003970471_details[class com.gigaspaces.grid.lookup.ServiceGridRegistrar], MaxConnPool: 1024 ]]]
java.rmi.ConnectException: Connect Failed to [NIO://YoramMacBookPro.local:53965/pid[63802]/1026921312924794_2_-519568947003970471_details[class com.gigaspaces.grid.lookup.ServiceGridRegistrar]]; nested exception is: 
	java.io.IOException: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused
	at com.gigaspaces.lrmi.nio.CPeer.connectAsync(CPeer.java:249)
	at com.gigaspaces.lrmi.nio.CPeer.connect(CPeer.java:195)
	at com.gigaspaces.lrmi.ConnectionPool.getConnection(ConnectionPool.java:110)
	at com.gigaspaces.lrmi.ConnPoolInvocationHandler.invoke(ConnPoolInvocationHandler.java:58)
	at com.gigaspaces.lrmi.MethodCachedInvocationHandler.invoke(MethodCachedInvocationHandler.java:76)
	at com.gigaspaces.lrmi.DynamicSmartStub.invokeRemote(DynamicSmartStub.java:457)
	at com.gigaspaces.lrmi.DynamicSmartStub.invoke(DynamicSmartStub.java:437)
	at com.gigaspaces.reflect.$GSProxy0.lookup(Unknown Source)
	at com.sun.jini.reggie.RegistrarProxy.lookup(RegistrarProxy.java:137)
	at net.jini.lookup.ServiceDiscoveryManager.lookup(ServiceDiscoveryManager.java:2803)
	at com.j_spaces.core.client.LookupFinder.find(LookupFinder.java:197)
	at com.j_spaces.core.client.LookupFinder.find(LookupFinder.java:123)
	at com.j_spaces.core.client.SpaceFinder.findJiniSpace(SpaceFinder.java:580)
	at com.j_spaces.core.client.SpaceFinder.findService(SpaceFinder.java:438)
	at com.j_spaces.core.client.SpaceFinder.find(SpaceFinder.java:313)
	at com.gigaspaces.client.SpaceProxyFactory.createSpaceProxy(SpaceProxyFactory.java:100)
	at org.openspaces.core.space.InternalSpaceFactory.create(InternalSpaceFactory.java:126)
	at org.openspaces.core.space.SpaceProxyFactoryBean.doCreateSpace(SpaceProxyFactoryBean.java:47)
	at org.openspaces.core.space.AbstractSpaceFactoryBean.afterPropertiesSet(AbstractSpaceFactoryBean.java:200)
	at org.openspaces.core.space.SpaceProxyConfigurer.createSpace(SpaceProxyConfigurer.java:48)
	at org.openspaces.core.space.AbstractSpaceConfigurer.create(AbstractSpaceConfigurer.java:31)
	at org.openspaces.core.space.AbstractSpaceConfigurer.space(AbstractSpaceConfigurer.java:37)
	at org.openspaces.core.GigaSpaceConfigurer.<init>(GigaSpaceConfigurer.java:89)
	at com.gigaspaces.demo.kstreams.app.DataGridTest.getOrCreateSpace(DataGridTest.java:204)
	at com.gigaspaces.demo.kstreams.app.DataGridTest.main(DataGridTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:282)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused
	at com.gigaspaces.lrmi.nio.CPeer.createAsyncChannel(CPeer.java:281)
	at com.gigaspaces.lrmi.nio.CPeer.connectAsync(CPeer.java:225)
	... 30 more
Caused by: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused
	at com.gigaspaces.async.SettableFuture.getResult(SettableFuture.java:59)
	at com.gigaspaces.async.internal.AbstractFuture.get(AbstractFuture.java:66)
	at com.gigaspaces.lrmi.nio.CPeer.createAsyncChannel(CPeer.java:275)
	... 31 more
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at com.gigaspaces.lrmi.nio.selector.handler.client.Conversation.finishConnection(Conversation.java:129)
	at com.gigaspaces.lrmi.nio.selector.handler.client.Conversation.handleKey(Conversation.java:89)
	at com.gigaspaces.lrmi.nio.selector.handler.client.ClientConversationRunner.doSelect(ClientConversationRunner.java:76)
	at com.gigaspaces.lrmi.nio.selector.handler.client.ClientConversationRunner.run(ClientConversationRunner.java:58)
	... 1 more

Apr 19, 2020 3:54:33 PM net.jini.lookup.ServiceDiscoveryManager lookup
INFO: Exception occurred during query, discarding reggie [com.sun.jini.reggie.RegistrarProxy[registrar=bdd1abdf-ef92-4c2f-b5f9-208cdf9c1a31 DynamicSmartStub [ImplObjClass: com.gigaspaces.grid.lookup.ServiceGridRegistrar, ConnectionURL: NIO://YoramMacBookPro.local:53965/pid[63802]/1026921312924794_2_-519568947003970471_details[class com.gigaspaces.grid.lookup.ServiceGridRegistrar], MaxConnPool: 1024 ]]]
java.rmi.ConnectException: Connect Failed to [NIO://YoramMacBookPro.local:53965/pid[63802]/1026921312924794_2_-519568947003970471_details[class com.gigaspaces.grid.lookup.ServiceGridRegistrar]]; nested exception is: 
	java.io.IOException: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused
	at com.gigaspaces.lrmi.nio.CPeer.connectAsync(CPeer.java:249)
	at com.gigaspaces.lrmi.nio.CPeer.connect(CPeer.java:195)
	at com.gigaspaces.lrmi.ConnectionPool.getConnection(ConnectionPool.java:110)
	at com.gigaspaces.lrmi.ConnPoolInvocationHandler.invoke(ConnPoolInvocationHandler.java:58)
	at com.gigaspaces.lrmi.MethodCachedInvocationHandler.invoke(MethodCachedInvocationHandler.java:76)
	at com.gigaspaces.lrmi.DynamicSmartStub.invokeRemote(DynamicSmartStub.java:457)
	at com.gigaspaces.lrmi.DynamicSmartStub.invoke(DynamicSmartStub.java:437)
	at com.gigaspaces.reflect.$GSProxy0.lookup(Unknown Source)
	at com.sun.jini.reggie.RegistrarProxy.lookup(RegistrarProxy.java:137)
	at net.jini.lookup.ServiceDiscoveryManager.lookup(ServiceDiscoveryManager.java:2803)
	at com.j_spaces.core.client.LookupFinder.find(LookupFinder.java:197)
	at com.j_spaces.core.client.LookupFinder.find(LookupFinder.java:123)
	at com.j_spaces.core.client.SpaceFinder.findJiniSpace(SpaceFinder.java:580)
	at com.j_spaces.core.client.SpaceFinder.findService(SpaceFinder.java:438)
	at com.j_spaces.core.client.SpaceFinder.find(SpaceFinder.java:313)
	at com.gigaspaces.client.SpaceProxyFactory.createSpaceProxy(SpaceProxyFactory.java:100)
	at org.openspaces.core.space.InternalSpaceFactory.create(InternalSpaceFactory.java:126)
	at org.openspaces.core.space.SpaceProxyFactoryBean.doCreateSpace(SpaceProxyFactoryBean.java:47)
	at org.openspaces.core.space.AbstractSpaceFactoryBean.afterPropertiesSet(AbstractSpaceFactoryBean.java:200)
	at org.openspaces.core.space.SpaceProxyConfigurer.createSpace(SpaceProxyConfigurer.java:48)
	at org.openspaces.core.space.AbstractSpaceConfigurer.create(AbstractSpaceConfigurer.java:31)
	at org.openspaces.core.space.AbstractSpaceConfigurer.space(AbstractSpaceConfigurer.java:37)
	at org.openspaces.core.GigaSpaceConfigurer.<init>(GigaSpaceConfigurer.java:89)
	at com.gigaspaces.demo.kstreams.app.DataGridTest.getOrCreateSpace(DataGridTest.java:204)
	at com.gigaspaces.demo.kstreams.app.DataGridTest.main(DataGridTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:282)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused
	at com.gigaspaces.lrmi.nio.CPeer.createAsyncChannel(CPeer.java:281)
	at com.gigaspaces.lrmi.nio.CPeer.connectAsync(CPeer.java:225)
	... 30 more
Caused by: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused
	at com.gigaspaces.async.SettableFuture.getResult(SettableFuture.java:59)
	at com.gigaspaces.async.internal.AbstractFuture.get(AbstractFuture.java:66)
	at com.gigaspaces.lrmi.nio.CPeer.createAsyncChannel(CPeer.java:275)
	... 31 more
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at com.gigaspaces.lrmi.nio.selector.handler.client.Conversation.finishConnection(Conversation.java:129)
	at com.gigaspaces.lrmi.nio.selector.handler.client.Conversation.handleKey(Conversation.java:89)
	at com.gigaspaces.lrmi.nio.selector.handler.client.ClientConversationRunner.doSelect(ClientConversationRunner.java:76)
	at com.gigaspaces.lrmi.nio.selector.handler.client.ClientConversationRunner.run(ClientConversationRunner.java:58)
	... 1 more

